opencti-platform/opencti-front/src/utils/Charts.js

```diff
@@ -120,8 +150,38 @@ export const areaChartOptions = (
   chart: {
     type: 'area',
     background: 'transparent',
+    events: {
+      mouseMove(e) {
+        e.fromElement.style.cursor = 'pointer';
+      },
+    },
     toolbar: {
-      show: false,
+      show: true,
+      tools: {
+        download: true,
+        zoom: false,
+        zoomin: false,
+        zoomout: false,
+        pan: false,
+        reset: false,
+      },
+      export: {
+        csv: {
+          filename: undefined,
+          columnDelimiter: ',',
+          headerCategory: 'category',
+          headerValue: 'value',
+          dateFormatter(timestamp) {
+            return new Date(timestamp).toDateString();
+          },
+        },
+        svg: {
+          filename: undefined,
+        },
+        png: {
+          filename: undefined,
+        },
+      },
```

Comment:
-You should probably add some [legend](https://apexcharts.com/docs/options/legend/) options to get the full png and svg export? not sure (for now export are uncomplete)

Reply:
The legend options has nothing to do to have the full png or svg export :) But thanks for your research !